dask/dataframe/tests/test_categorical.py

```diff
@@ -266,6 +266,30 @@ def test_categorical_set_index(shuffle):
         assert list(sorted(d2.index.compute())) == ["b", "b", "c"]
 
 
+@pytest.mark.parametrize(
+    "ncategories", list(range(1, 6)), ids=lambda x: f"{x} categories"
+)
+@pytest.mark.parametrize(
+    "npartitions", list(range(1, 6)), ids=lambda x: f"{x} partitions"
+)
```

Comment:
In the interests of testing time, I suspect that we could reduce the matrix here from 25 tests to a smaller number, while still testing what we're after.  Maybe `[1, 3, 6]` instead of `range(1, 6)`?

Reply:
Good point, fixed!